### ISSUE TITLE
Support for Swift 4.2

### ISSUE THREAD

**Issue by WilsonGramer** [alldritt/EurekaColorPicker#9]
Many properties have been renamed in Swift 4.2, making ColorPickerRow unbuildable.

**Comment by alldritt** [comment]
The Swift4.2 branch contains changes to address Swift 4.2 issues and update to Eureka 4.3.  I'll merge this into the master branch in a while once I'm sure its all working as it should.

You can use this URL in your Podfile:

```
pod 'ColorPickerRow', :git => 'https://github.com/EurekaCommunity/ColorPickerRow', :branch => 'Swift4.2'
```

**Comment by alldritt** [comment]
I have merged Swift 4.2 support into the master branch.  Use version 1.3.0 or later for Swift 4.2.
